src/sample-account-setup.test.ts

```diff
@@ -1,27 +1,546 @@
+  );
+
+  const respiratoryRate = await medplum.createResourceIfNoneExist<Observation>(
+      {
+        resourceType: 'Observation',
+        id: 'respiratory-rate',
+        subject,
+        code: {
+          coding: [
+            {
+              code: '9279-1',
+              display: 'Respiratory rate',
+              system: 'http://loinc.org',
+            },
+          ],
+          text: 'Respiratory rate',
+        },
+        valueQuantity: {
+          code: '/min',
+          system: 'http://unitsofmeasure.org',
+          unit: '/min',
+          value: 15,
+        },
+        effectiveDateTime: date.toISOString(),
+        status: 'final',
+      },
+      'Observation/respiratory-rate'
+  );
+
+  const heartRate = await medplum.createResourceIfNoneExist<Observation>(
+      {
+        resourceType: 'Observation',
+        id: 'heart-rate',
+        subject,
+        code: {
+          coding: [
+            {
+              code: '8867-4',
+              display: 'Heart rate',
+              system: 'http://loinc.org',
+            },
+          ],
+          text: 'Heart rate',
+        },
+        valueQuantity: {
+          code: '/min',
+          system: 'http://unitsofmeasure.org',
+          unit: '/min',
+          value: 80,
+        },
+        effectiveDateTime: date.toISOString(),
+        status: 'final',
+      },
+      'Observation/heart-rate'
+  );
+
+  const weight = await medplum.createResourceIfNoneExist<Observation>(
+      {
+        resourceType: 'Observation',
+        id: 'weight',
+        subject,
+        code: {
+          coding: [
+            {
+              code: '29463-7',
+              display: 'Body Weight',
+              system: 'http://loinc.org',
+            },
+          ],
+          text: 'Body Weight',
+        },
+        valueQuantity: {
+          code: 'kg',
+          system: 'http://unitsofmeasure.org',
+          unit: 'kg',
+          value: 70,
+        },
+        effectiveDateTime: date.toISOString(),
+        status: 'final',
+      },
+      'Observation/weight'
+  );
+
+  //Make a default message "Hello and welcome to our practice"
+  const message = await medplum.createResourceIfNoneExist<Communication>(
+      {
+        resourceType: 'Communication',
+        id: 'welcome-message',
+        subject,
+        sender: subject,
+        payload: [
+          {
+            contentString: 'Hello and welcome to our practice',
+          },
+        ],
+      },
+      'Communication/welcome-message'
+  );
 
-  expect(handler(medplum, { input, contentType })).resolves.toBe(true);
-});
+  return {
+    patient,
+    practitioner,
+    schedule,
+    slots,
+    hemoglobinA1c,
+    activeCarePlan,
+    completedCarePlan,
+    diagnosticReport,
+    activeMedication,
+    stoppedMedication,
+    completedImmunization,
+    notDoneImmunization,
+    bloodPressure,
+    bodyTemperature,
+    height,
+    respiratoryRate,
+    heartRate,
+    weight,
+    message
+  };
+}
```

Comment:
@codyebberson and @rahul1 - this part is interesting.  Not sure the right path here, but this seems sub-optimal